src/patternfly/layouts/Grid/examples/grid-responsive-gutters-example.hbs

```diff
@@ -0,0 +1,62 @@
+{{#> grid grid--modifier="pf-m-gutter pf-m-gutter-3xl-on-lg"}}
```

Comment:
Why do you need to set `pf-m-gutter pf-m-gutter-3xl-on-lg' twice

Reply:
That will apply `.pf-m-gutter` spacing (lg) by default, then `.pf-m-gutter-3xl` spacing on the `lg` viewport. Or are you asking something else?

Reply:
I guess where I'm getting confused is that we have a default gutter size of lg. why do we need a default and a lg?

Reply:
We can't get rid of `.pf-m-gutter` since that would be a breaking change. We could leave out `.pf-m-gutter-lg`, though do you think it would be weird to offer every size except for `lg`? Also, the spacing for `$pf-global--gutter` might change at some point, so `.pf-m-gutter` would always represent the value of our global gutter value, so I can see value in leaving `.pf-m-gutter` as separate from the explicit widths. What do you think?

Reply:
I'm in favor of removing pf-m-gutter next breaking change period and removing it from examples now.
I don't think we need both and I can't think of anywhere else across the system where when we give options for a range of sizes, that we have had one default value, and then repeated that default value within the range of the sizes. i.e. Table has grid breakpoint modifiers but no default grid breakpoint, our utilities don't have one default.

Reply:
An argument for leaving it would be for consistency in implementation - out of the box you get our default gutter. Otherwise it would be up to apps to choose any time they want a gutter, when I presume most will just want some reasonable default like `md` or `lg`. Also what about split, stack, and gallery, which all use `$pf-global--gutter`, too?

Reply:
I'd also be in favor of getting rid of the default variable then just assigning one of the sized variables to default, I assume it will be less confusing than having a default var which also happens to me -med for example.

Reply:
All valid points. It makes sense to me to set `pf-m-gutter` as a default and not include breakpoints for that selector. So, non-responsive support for default `pf-m-gutter`. Then if gutters are being changed responsively, a size would be expected. Yes, `pf-m-gutter-lg` would be redundant, but only once for the non-responsive version.

**Example**
```
<div class="pf-l-grid pf-m-gutter"></div>
```
vs (what I expect to be the most common use case)
```
<div class="pf-l-grid pf-m-gutter pf-m-gutter-xl-on-xl"></div>
```
or
```
<div class="pf-l-grid pf-m-gutter-sm pf-m-gutter-xl-on-xl"></div>
```

Reply:
Personally I'm in favor of either removing the default `pf-m-gutter` or leaving it along with all of the breakpoints. @mattnolting I can see a use case of someone wanting like `.pf-m-gutter.pf-m-gutter-none-on-md.pf-m-gutter-on-lg`, and I think it might be confusing to offer specific gutter sizes with breakpoints without the default gutter supporting breakpoints. I don't see why we wouldn't want to offer that aside from it being a few more lines of code.

And I'm more in favor of leaving `pf-m-gutter` as the default gutter and including in the documentation that it is the same as `pf-m-gutter-lg`. So you can either use `gutter` (with all of the breakpoints) for a default gutter, or `gutter-[size]` for a custom gutter. But I'm also fine dropping `pf-m-gutter`, too. 

That said, I think we should add this same option to the split, stack, and gallery layouts (or anywhere we offer a generic layout with the option of a gutter - unless anyone disagrees?), and removing `.pf-m-gutter` will probably be a fairly invasive change since that would require folks define a gutter size where they're just using the default gutter currently... unless we offer this in a backwards compatible way like leaving it in the code and not documenting it.

Another argument in favor of leaving the default gutter and its breakpoints is to offer the most flexibility to the user and encourage them to use it correctly in the documentation.

This also begs the question that if we drop `pf-m-gutter`, do we need `$pf-m-global--gutter`? Was that ever needed in the first place?